src/Dumplie/Domain/SharedKernel/Money/Currencies.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\SharedKernel\Money;
+
+final class Currencies
```

Comment:
To be honest there is no good reason, but currencies are not a kind of data that change every day or even month, we need to store them somewhere anyway